tests/playwright/sanity/modules/nested-tabs/nested-tabs.test.ts

```diff
@@ -1144,12 +1144,14 @@ test.describe( 'Nested Tabs tests @nested-tabs', () => {
 		await setup( wpAdmin );
 		const editor = await wpAdmin.useElementorCleanPost(),
 			container = await editor.addElement( { elType: 'container' }, 'document' ),
-			frame = await editor.getPreviewFrame();
+			frame = await editor.getPreviewFrame(),
+			nestedTabsWidget = await frame.locator( 'e-n-tabs' ).first();
 
 		// Add widget.
 		await editor.addWidget( 'nested-tabs', container );
 		Array.from( { length: 7 }, async () => {
 			await page.locator( 'div:nth-child(2) > .elementor-repeater-row-tools > button:nth-child(2)' ).click();
+			await editor.isUiStable( nestedTabsWidget, 3, 200 );
```

Comment:
Looks like duplication of the tab times out in the loop because it loops before the previous action completed in the editor.